src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import cn from 'classnames';
+import { FilterBy } from '../../types/FilterBy';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onFilterBy: (filter: FilterBy) => void,
+  filterBy: FilterBy,
+  onClearCompleted: (id: number) => void,
+};
+
+export const Footer: React.FC<Props> = React.memo(({
+  todos,
+  onFilterBy,
+  filterBy,
+  onClearCompleted,
+}) => {
+  const completedTodos = todos.filter((todo) => todo.completed);
+  const activeTodos = todos.filter((todo) => !todo.completed);
+  const handleClearCompleted = () => {
```

Comment:
what is the purpose if this function? it doesn't return anything, and doesn't do anything, check it